### ISSUE TITLE
Fraction bar from latex is not appearing in copied images

### ISSUE THREAD

**Issue by adamtwest** [brendena/MathEquations-Extension#3]
Great project, thanks for making! Fraction bar is not showing up in copied images from Latex equations, e.g.,  A=B\frac{C_1C_2}{D^2} or \frac{3}{4}. Shows up in preview, but not in image pasted into Google Docs.

**Comment by brendena** [comment]
I'm glad you like it.  I've been working on a big update for this extension
for a while, but there are a few big bugs that are preventing me from
publishing them.  But i'm hoping in a couple of weeks i'll be able to push
up and they should fix your problems.

**Comment by Xorinet** [comment]
I agree, this is great! Sadly, the problem still exists. Is there any way you can fix this?

**Comment by isaacagudo** [comment]
Same issue here with Google Chrome and fractions bars. Could you please give us some hints on how to solve the problem? We might work on a PR ...
